packages/sdk/contractkit/src/base.ts

```diff
@@ -31,6 +32,13 @@ export enum CeloContract {
 
 export type StableTokenContract = CeloContract.StableToken | CeloContract.StableTokenEUR
 
+type TokenKeys = Record<string, StableTokenContract> // this is probably not the way to do this
+
+export const StableToken: TokenKeys = {
+  cUSD: CeloContract.StableToken,
+  cEUR: CeloContract.StableTokenEUR,
+}
+
```

Comment:
I need an object (or enum), that I can call `StableTokens.cUSD` or `StableTokens.cEUR` and have it's own type. (only ones I could find the keys were `CeloContract.StableToken` and `CeloContract.StableTokenEUR`)

I also would like this objects to be iterable (or can call a map).

Reply:
Check out https://github.com/celo-org/celo-monorepo/blob/master/packages/sdk/contractkit/src/celo-tokens.ts#L7

And then you can call this to get the contract: https://github.com/celo-org/celo-monorepo/blob/master/packages/sdk/contractkit/src/celo-tokens.ts#L233

Not sure if you've poked around that file in general but I think some of the code in there would be useful